<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionWindow</class>
 <widget class="QWidget" name="ConnectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="2">
      <widget class="QLineEdit" name="portLineEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="hostLabel">
       <property name="text">
        <string>Server name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Server port</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="5">
      <widget class="QDialogButtonBox" name="buttonBox"/>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QComboBox" name="hostCombo">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="ethernetRadioButton">
       <property name="text">
        <string>Ethernet</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
